moveit_planners/chomp/chomp_motion_planner/include/chomp_motion_planner/chomp_optimizer.h

```diff
@@ -84,25 +84,21 @@ class ChompOptimizer
   inline double getPotential(double field_distance, double radius, double clearence)
   {
     double d = field_distance - radius;
-    double potential = 0.0;
 
-    // three cases below:
-    if (d >= clearence)
+    if (d >= clearence)  // everything is fine
     {
-      potential = 0.0;
+      return 0.0;
     }
-    else if (d >= 0.0)
+    else if (d >= 0.0)  // transition phase, no collision yet
     {
       double diff = (d - clearence);
-      double gradient_magnitude = diff * clearence;  // (diff / clearance)
-      potential = 0.5 * gradient_magnitude * diff;
+      double gradient_magnitude = diff / clearence;
```

Comment:
I'd use consts here as much as possible but it's really nitpicking so please feel free to resolve & ignore if you don't want to make those changes

Reply:
Would be fine with me. Next time, please propose such a small change directly in github. This allows me to accept it and proceed.

Reply:
I think, this is ready to merge then. @bmagyar, can you approve?

Reply:
Righty-ho captain!